src/main/java/com/agilethought/atsceapi/exception/ErrorMessage.java

```diff
@@ -5,6 +5,10 @@
     public static final String BAD_REQUEST_MESSAGE = "The field ";
     public static final String BAD_REQUEST_MESSAGE_TYPE = "needs to be 1 or 2";
     public static final String BAD_REQUEST_MESSAGE_FIRST_NAME_LASTNAME = "is wrong because you need to type it";
-    public static final String BAD_REQUEST_MESSAGE_EMAIL_PASSWORD = "is wrong because it doesn't have the correct format";
```

Comment:
Your current changes are fine, but I want to take advantage of your current PR, to make a simple request. Do you think you can change those 2 messages? 
If I'm not mistaken those errors are sent when the user sends upper case letters and you only want them to input lowercase ones, correct? 
If that's the case I think you should try to be more clear, like "Name fields should be written with lower case letters only", for example. Same with email.

Reply:
Actually, we can receive letters in uppercase in the firstname and lastname, that's why I delete the method that check if the string is in lowercase.
We can receive uppercase letter in the email and password, that´s why I spare the constant 
`BAD_REQUEST_MESSAGE_EMAIL_PASSWORD` in two constants to have a better undestanding of what could faild

Reply:
I see, but in that case when do those 2 messages pop up then? When the user leaves those fields blank? 
My problem with those messages is that they aren't descriptive enough I don't know what they mean when I read them. So my request is "Change them to something more specific and easy to understand"

Reply:
I upload the messages, let me know if they are good or if I need to improve the messages